### ISSUE TITLE
:octocat: Everyone is welcome to contribute 💯 

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#2026]
PyTorch-Ignite is a community effort, and everyone is welcome to contribute and join the contributors community no matter your background and skills ! 

If this is something you would like to do: help the project and join the community, please take a look at our [contributing guide](https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md) and ask any related questions here or on our [Discord server](https://discord.com/invite/djZtm3EmKj).

We also has a list of issue we would like to ask for a help:
Good for beginners:
- https://github.com/pytorch/ignite/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
This list assumes some knowledge about the library, training neural networks
- https://github.com/pytorch/ignite/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

**Comment by sahilg06** [comment]
Hi @vfdev-5, I am new to PyTorch ignite. I would like to contribute to it. Can you please guide me to some **good first issue**? 
Thanks!

**Comment by vfdev-5** [comment]
@sahilg06 thanks for asking ! There is a new good first issue to fix mypy failure: https://github.com/pytorch/ignite/issues/2046 
Please comment out there if you would like to tackle it.

**Comment by anirudhb11** [comment]
Hi @vfdev-5, I am also new to Pytorch Ignite and would love to contribute to it. Can you please guide me to some good first issue?
Thanks!

**Comment by harshit2000** [comment]
Hi @vfdev-5, I am new to Pytorch Ignite and the project seems interesting. I am learning via docs and want to work on my first good issue. Can you highlight any?

**Comment by Jacob208M** [comment]
Hi @vfdev-5. I would like to contribute to Pytorch Ignite. Can you assign me too for good first issue to work with?

**Comment by vfdev-5** [comment]
@Jacob208M thanks for asking. Here is a recent "help wanted" feature request we have : https://github.com/pytorch/ignite/issues/2703 that you can try to tackle. Please also consider to join our Discord to ask questions and get quick answers: https://pytorch-ignite.ai/chat

**Comment by harry7337** [comment]
@vfdev-5 Hey! I was looking for a few first issues to contribute to! Can you guide me to a good first issue? Cause it seems the ones with the labels already have people working on them.

**Comment by vfdev-5** [comment]
@harry7337 If you see an open issue that is open for more than 1 week and even if someone pretended to work on that, you can work on that as it most probably no one is working on that...

**Comment by Abhishek-kumar0503** [comment]
Hi @vfdev-5, I am new to Pytorch Ignite and the project seems interesting. So,I want to work on my first good issue. Can you assign me too for good first issue to work with?

**Comment by vfdev-5** [comment]
Hey @anirudhb11 , glad that you find the project interesting. 

> So,I want to work on my first good issue. Can you assign me too for good first issue to work with?

This can depend on your skills. Let's try this issue: https://github.com/pytorch/ignite/issues/2791 . So, read the description of the issue and try to understand what we are trying to solve there and propose your solution. Please ask other related questions if any in `#start-contributing` channels on discord (I saw you already joined it).

**Comment by BabaYaga1221** [comment]
hey @vfdev-5, I am new to the open-source project community and I am interested to contribute on this project. So, I want to work on some good first issue. Can you help me and assign the issue to work? Its a great opportunity for me to contribute in this project. Thanks @vfdev-5

**Comment by vfdev-5** [comment]
@BabaYaga1221 as you are new to open-source, check https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md
then also check the website: https://pytorch-ignite.ai/
As for good first issues, you can go through the docs: website and API: https://pytorch.org/ignite/ and search for typos and unclear explanations. Once found something, you can create an issue with "docs" label and report the finding.

**Comment by Somarfagell** [comment]
Hi @vfdev-5, I'm also new into the open source waters, I really like the project, Is there any good issue to begin with?.

**Comment by vfdev-5** [comment]
Hey @Somarfagell, you can check this issue if you would like: https://github.com/pytorch/ignite/issues/2917
Otherwise all previous comments are also partly applicable.

**Comment by looksy811** [comment]
Hi, @vfdev-5, I'm interested to contribute on this project.  But I've never tried OSS contributor.  Can you assign me too for good first issue ??

**Comment by GuptaPratik02** [comment]
Hi @vfdev-5 , I am new to PyTorch ignite. I am a Python Developer and would like to contribute in this, any bug fix or any other contributions required would work. Can you please guide me to work on some good first issue?
Your help will be great for me to get started with the issue.
Thanks!

**Comment by avinashsharma080** [comment]
Hi @vfdev-5 I am new to PyTorch ignite and want to contribute. It will be great if I start my open-source contribution journey from here. So if there is any good first issue, I would like to get started with it.

Thanks in advance!

**Comment by M0I0A** [comment]
Hi @vfdev-5, I am new to Pytorch Ignite and the project seems interesting. So,I want to work on my first good issue. Can you assign me too for good first issue to work with?

**Comment by aaryanraj2120** [comment]
Hi @vfdev-5  I am new to PyTorch ignite and want to contribute. It will be great if I start my open-source contribution journey from here. So if there is any good first issue, I would like to get started with it.

Thanks in advance!

**Comment by vfdev-5** [comment]
@aaryanraj2120 you just copied the message of [avinashsharma080](https://github.com/avinashsharma080) :)

**Comment by Muhammad-Rebaal** [comment]
Hi @vfdev-5 !
I'm new here in the Pytorch Ignite community !
I'm an associate software engineer and love to do ml stuff in python . So, is there any issue regarding that on which I can contribute . So, please let me know ?
ThankYou !

**Comment by vfdev-5** [comment]
Hi @Muhammad-Rebaal , check this one: https://github.com/pytorch/ignite/issues/3335

**Comment by Dronten** [comment]
Hey , I m new here and I would love to contribute in this , can i have a little guidance on it

**Comment by vfdev-5** [comment]
@Dronten welcome! Check this issue: https://github.com/pytorch/ignite/issues/3335 which should be rather accessible to everyone.

**Comment by dxvidparham** [comment]
Hey everyone!

Are there any other issues than #3335 that newcomers can currently work on :)?

**Comment by vfdev-5** [comment]
@dxvidparham you can filter out the issues by good first issue label: https://github.com/pytorch/ignite/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22

**Comment by mariam-29** [comment]
hey @vfdev-5 , I am new to open source, I am into ML/AI. Can you assign me  for good first issue ??

**Comment by phanr8541** [comment]
Hi @vfdev-5 , I'm new to open source and learning for one of my classroom course. Is there any simple documentation I can contribute to for a learning experience

**Comment by maifeeulasad** [comment]
<img width="1301" height="477" alt="Image" src="https://github.com/user-attachments/assets/df11660a-4619-4c64-b903-4b27d571434b" />

Nothing left to contribute! :(

I was hoping to contribute!

**Comment by vfdev-5** [comment]
> Nothing left to contribute! :(
> 
> I was hoping to contribute!

You can check this one: https://github.com/pytorch/ignite/issues/3481

**Comment by maifeeulasad** [comment]
Really appreciate it @vfdev-5 😍

**Comment by Perkyenmity** [comment]
@vfdev-5 Hello there. I am new to open source and I want to contribute to pytorch-ignite. I am familiar with python basics and also have an understanding of version control systems like git and github. I will go through the contributing guide and the Readme.md file of this project and try to understand the codebase and get back to you. Please guide me through my first contribution for this organization. Thanks

**Comment by Shubham91999** [comment]
Hey, there! @vfdev-5. I am new to open source, looking forward to contributing. Are there any good first issues left?

**Comment by mahakal45x09** [comment]
Hi, I’m a new contributor interested in working on this issue. I have experience with Python and PyTorch and would like to attempt this task if it’s still available. @vfdev-5

**Comment by 21f3000703** [comment]
Hi, Ii would like to work on it @vfdev-5

**Comment by nithish-74** [comment]
Hi, I’m new to PyTorch Ignite and would like to start contributing.
I have experience with Python and basic PyTorch, and I’m currently preparing a GSoC proposal related to training workflows.
Could you please suggest a good first issue that would be a suitable starting point for me @vfdev-5

**Comment by Rishiii57** [comment]
Hello @vfdev-5 , I want to start contributing. Is there anything where I can help?

**Comment by Bubu02** [comment]
Hey @vfdev-5 👋🏻, thank you for creating this issue and keeping the community so welcoming to new contributors!
​I’m incredibly passionate about this field and curious to learn more, but to be honest, I'm pretty new to open-source contributions. I’d love to get involved and help out with PyTorch-Ignite. Could you point me toward a good first issue from the list that would be a great starting point? Thanks so much! 🤝🏻😄

**Comment by SaranshDhage** [comment]
Hi @vfdev-5, I'm an AI/ML engineer interested in contributing. Are there any open good first issues available? Keen to contribute.

**Comment by affanhamid** [comment]
Hi @vfdev-5 I'm new to ignite and I wish to start contributing. Can you guide me to a good first issue I can tackle?

Thank you

**Comment by MelosaRao** [comment]
Hi @vfdev-5, thank you for maintaining this page. I am new to open source contributing, but I have worked with Python and Pytorch on student projects. Could you suggest a good first issue?

**Comment by SreyasM24** [comment]
Hi everyone! I'm interested in contributing to PyTorch-Ignite. I've just finished setting up the project. Any tips for a beginner before I pick my first issue?

**Comment by 01023833096ks-tech** [comment]
Hi @vfdev-5, I am new to PyTorch ignite. I would like to contribute to it. Can you please guide me to some good first issue?
Thanks!

**Comment by Arun-sunny23** [comment]
Hi everyone! 👋 Just merged my first PR to pytorch-ignite and would love to keep contributing regularly. @vfdev-5 any suggestions on what to look at next?

**Comment by mimihere33** [comment]
Hii @vfdev-5 I would love to start contributing. can you please suggest any good first issues?

**Comment by syimyq1** [comment]
Hello everyone! I am a beginner Python developer and I would love to make my first open-source contribution to PyTorch-Ignite. Could you please suggest a simple 'good first issue' that I can work on? I am familiar with Git and GitHub basics. Thank you!
